plugins/modules/ims_acb_gen.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) IBM Corporation 2020
+# Apache License, Version 2.0 (see https://opensource.org/licenses/Apache-2.0)
+
+from __future__ import absolute_import, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+DOCUMENTATION = r'''
+---
+module: ims_acb_gen
+short_description: Generate IMS ACB
+version_added: "2.9"
+description:
+  - The ims_acb_gen module generates an IMS application control block (ACB) necessary for an IMS application program to be scheduled and run. 
+  - The ims_dbd_gen and ims_psb_gen modules can be used to generate the associated IMS DBDs and PSBs to be used with the ims_acb_gen module. 
```

Comment:
please expand DBD and PSB at the first instance, just to be consistent with the ACB definition - line 21

Reply:
updated. Thanks Radha!